packages/ckeditor5-dev-env/lib/release-tools/utils/cli.js

```diff
@@ -34,7 +34,7 @@ const cli = {
 		};
 
 		return inquirer.prompt( [ confirmQuestion ] )
-			.then( answers => answers.confirm );
+			.then( answers => answers.confirm ? Promise.resolve() : Promise.reject() );
```

Comment:
I'll merge this PR anyway (if I won't find other issues). But please fix it anyway.